.github/workflows/focus-sync-strings.yml

```diff
@@ -2,45 +2,50 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/
 
-name: "Focus - Sync Strings"
+name: Focus - Uplift Strings
 
 on:
   schedule:
-    - cron: '0 2 * * *'
+    - cron: "0 2 * * *"
   workflow_dispatch:
 
 jobs:
   main:
-    name: "Sync Strings"
-    runs-on: ubuntu-20.04
+    name: Uplift strings
+    runs-on: ubuntu-latest
     steps:
-      - name: "Discover Focus Beta Version"
-        id: focus-android-beta-version
+      - name: Discover Focus beta version
+        id: discover-beta-version
         uses: mozilla-mobile/fenix-beta-version@4.2.2
-      - name: "Skip non-beta versions"
+      - name: Skip non-beta versions
         uses: andymckay/cancel-action@0.2
-        if: ${{ steps.focus-android-beta-version.outputs.beta_version == '' }}
-      - name: "Checkout Main Branch"
-        uses: actions/checkout@v2
+        if: ${{ steps.discover-beta-version.outputs.beta_version == '' }}
+      - name: Store ENV variables
+        id: env-variables
+        run: |
+          echo "project_name=Focus Strings" >> $GITHUB_OUTPUT
+          echo "beta_branch=releases_v${{ steps.discover-beta-version.outputs.beta_version }}" >> $GITHUB_OUTPUT
+          echo "branch_name=automation/uplift-focus-strings-${{ steps.discover-beta-version.outputs.beta_version }}" >> $GITHUB_OUTPUT
```

Comment:
I would consider dropping `project_name` and `branch_name` as variables from these changes. I think it adds unnecessary complexity to something rather simple since these variables are only ever used once and are pretty simple strings. `beta_branch` is at least used multiple times so that is fine.

Reply:
I personally don't think it adds complexity, and it puts all aspects specifics to a product in one place.
To adapt this to Fenix you would only need to update this section, instead of 2, and ending up with different versions in each place.

With that said, you're likely the one who needs to maintain these going forward, so I'll go with your preference.